### DISCUSSION TITLE
How to disable the `took` from prompt?

### DISCUSSION THREAD

**Discussion by AdsonCicilioti** [starship/starship#5577] [Q&A]
I'm using bash with ble.sh and starship on nixOS.  I don't know if it's something in my config, but I don't like the behavior of this time counter, as here it returns the time in which the shell was idle without executing a command.

 While what I was expecting is for it to return the execution time of a command.

 So I prefer to disable this.  Could anyone tell me how?

**Comment by mickimnet** [comment]
That is the module [Command Duration](https://starship.rs/config/#command-duration). You could create your own configuration, or just run `starship config cmd_duration.disabled true`, which will create a minimal configuration for you.

**Reply by AdsonCicilioti** [reply]
Right!
I have my `yaml` configuration file. What would be the parameter?

```yml
[cmd_duration]
disabled = true
```
??
